### DISCUSSION TITLE
Map Publishing

### DISCUSSION THREAD

**Discussion by AnushiyaBalasubramanian** [opengeos/leafmap#111] [Q&A] [answered]
Hello,
I have Installed Datapane. When I Publishing map using m.publish(name="--") . It Display AttributeError. How to solve it ?
Thanks.

**Comment by giswqs** [comment, marked as answer]
Try `leafmap.update_package()`

**Reply by AnushiyaBalasubramanian** [reply]
It's Working Sir.
Thanks.
